packages/components/src/components/TaskTree/TaskTree.js

```diff
@@ -23,9 +23,17 @@ class TaskTree extends Component {
 
   render() {
     const { selectedTaskId, taskRuns } = this.props;
+
+    if (!taskRuns) {
```

Comment:
ideally we'd catch this higher up the tree, but if we want to render nothing we can just return `null`

Reply:
Weird, with this and the below (index.js change) I can reliably get `Uncaught TypeError: steps.map is not a function`, keeping this unchanged for now I think 🤔